### PR TITLE
PMM-9321 verify absolute time range setting

### PR DESCRIPTION
**Pull request by ElenaFedichkina**
Verify absolute time range setting
https://pmm.cd.percona.com/job/pmm2-ui-test-nightly/940/
image

### CODE REVIEW

**Comment by yurkovychv** [comment]
@ElenaFedichkina your test is failing on CI. PTAL

**Review by yurkovychv — DISMISSED** [review]

File: tests/QAN/timerange_test.js [hunk 1]
```diff
@@ -39,6 +39,27 @@ Scenario(
   },
 );
 
+Scenario(
+  'PMM-T432 Open the QAN Dashboard and check that changing absolute time range updates the overview table, URL. @nightly @qan',
+  async ({ I, adminPage, qanDetails, qanFilters, qanOverview }) => {
+    const today = new Date();
+    const date = `${today.getFullYear()}-${today.getMonth() + 1}-${today.getDate()}`;
```
Comment (vyarosh) [inline tests/QAN/timerange_test.js:46]: we have moment lib added at some point. Do you consider using moment for given strings since we already have it?

**Review by vyarosh — COMMENTED** [review]
Left a comment for a 'moment' lib usage.

File: tests/QAN/timerange_test.js (hunk 1 above)
Comment (ElenaFedichkina) [reply tests/QAN/timerange_test.js:46]: thanks. i will use it

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**